<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransferOptionsClass</class>
 <widget class="QWidget" name="FileTransferOptionsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>69</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="chbAutoReceive">
     <property name="text">
      <string>Automatically receive files from contacts in roster</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbHideDialog">
     <property name="text">
      <string>Hide dialog after transfer started</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chbRemoveFinished">
     <property name="text">
      <string>Automatically remove finished transfers</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>chbAutoReceive</tabstop>
  <tabstop>chbHideDialog</tabstop>
  <tabstop>chbRemoveFinished</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
